Add browser table release changelog entry (#166)
broadinstitute/gnomad-blog: content/changelog/2024-08-release-gnomad-browser-tables.md

---
title: Release gnomAD browser tables

> date: 2024-08-20

Reviewer: Update date

---
title: Release gnomAD browser tables
date: 2024-08-20
order: 1
---

The Hail Tables that underlie the gnomAD browser are now available for public use. These tables closely mirror the structure of the browser, enabling users familiar with the gene and variant pages to easily access the full set of underlying data in a format consistent with the browser's organization.

For more information about these tables, please refer to our [help text](http://gnomad.broadinstitute.org/help/v4-browser-hts) on the browser release tables. 

To download these tables, visit our [downloads page](http://gnomad.broadinstitute.org/downloads#v4-browser-tables).

<!-- end_excerpt -->

> **Three tables are now available**

Reviewer 2: ```suggestion
**Three tables are now available:**
```
(optional)

> - v4.1 browser variants Hail Table

Reviewer 2: ```suggestion
- gnomAD v4.1 browser variants Hail Table
```

- GRCh38 browser gene models Hail Table
- GRCh37 browser gene models Hail Table

> ### gnomAD v4.1 Exome/Genome/Joint Variant Hail Table

Reviewer 2: ```suggestion
### gnomAD v4.1 browser variants Hail Table
```
for consistency with the bullet point above

> The browser variant table consolidates data from exomes, genomes, and joint frequencies into a single, unified table. Frequency information is stored as a dictionary, the key-value pairs allow for easy access to specific frequencies.

Reviewer 2: ```suggestion
The browser variant table consolidates data from the gnomAD [exome](https://gnomad.broadinstitute.org/downloads#v4-variants), [genome](https://gnomad.broadinstitute.org/downloads#v4-variants), and [joint frequency](https://gnomad.broadinstitute.org/downloads#v4-joint-freq-stats) releases into a single, unified table. Frequency information is stored as a dictionary, and the key-value pairs allow for easy access to specific frequencies.
```

> ### gnomAD GRCh38/GRCh37 Gene Model Hail Tables

Reviewer 2: ```suggestion
### GRCh38/GRCh37 Gene Models Hail Tables
```

> The gene models tables aggregate relevant information from multiple sources into a single table. For each gene, data is included from GENCODE, HGNC, MANE transcripts, GTEx, and various gnomAD secondary analyses.

Reviewer 2: ```suggestion
The gene models tables aggregate relevant information from multiple sources into a single table. For each gene, data is included from GENCODE, HGNC, and MANE transcripts. Information from GTEx, and various gnomAD secondary analyses are available in the GRCh37 table but are not yet available in the GRCh38 table.
```
